utils/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/KafkaTestParametersInfo.scala

```diff
@@ -0,0 +1,32 @@
+package pl.touk.nussknacker.engine.kafka.source
+
+import io.circe.Json
+import pl.touk.nussknacker.engine.api.definition.Parameter
+import pl.touk.nussknacker.engine.util.json.BestEffortJsonEncoder
+
+case class KafkaTestParametersInfo(uiParameters: List[Parameter], schemaIntId: Option[Int], formatMessage: Any => Json)
+
+object KafkaTestParametersInfo {
+
+  def empty: KafkaTestParametersInfo = KafkaTestParametersInfo(Nil, None, _ => Json.Null)
+
+  def wrapWithConsumerData(testData: Json, schemaIntId: Option[Int], topic: String): Json = {
+    BestEffortJsonEncoder.defaultForTests.encode(
```

Comment:
We have to make these fields connected with the test data reading mechanism. Have you got a plan for that?

Reply:
:thinking: Not yet but I will take a look on that one.